plugins/database/plugin.go

```diff
@@ -77,6 +78,17 @@ func configure(_ *node.Plugin) {
 		log.Fatalf("Failed to check database version: %s", err)
 	}
 
+	if str := config.Node().GetString(CfgDatabaseDirty); str != "" {
+		val ,err := strconv.ParseBool(str)
+		if err != nil {
+			log.Warnf("Invalid %s: %s", CfgDatabaseDirty, err)
+		} else if val  {
+			MarkDatabaseUnhealthy()
+		} else {
+			MarkDatabaseHealthy()
+		}
```

Comment:
Maybe this could be nested. Makes it a bit easier to read.

Reply:
I think what is there currently is better than this nested version.

````
if err != nil {
    log.Warnf("Invalid %s: %s", CfgDatabaseDirty, err)
} else  {
    if val  {
        MarkDatabaseUnhealthy()
    } else {
        MarkDatabaseHealthy()
    }
}